lib/datadog/core/remote/configuration/repository.rb

```diff
@@ -90,8 +91,12 @@ def initialize(repository)
             def contents_to_config_states(contents)
               return [] if contents.empty?
 
-              contents.map do |content|
-                {
+              contents.each_with_object([]) do |content, acc|
```

Comment:
I had to change it to `each_with_object` because steep was complaining about using `map` and `next`